### DISCUSSION TITLE
how can i get logits of qwen2.5's output ?

### DISCUSSION THREAD

**Discussion by joyyyhuang** [QwenLM/Qwen3#1063] [Q&A]
I am doing a classification task and want qwen to output only 0 or 1. How can I get the logits scores of these two tokens?
